console/react/src/connect/connectPage.js

```diff
@@ -82,7 +82,7 @@ class ConnectPage extends React.Component {
           </TextContent>
           <TextContent>
             <Text component="h2" className="connect-description">
-              The console displays information about a qpid dispatch router network. It
+              The console displays information about a Qpid Dispatch Router network. It
```

Comment:
@pwright This is where I wasn't sure about "router". Should the sentence be

* displays information about a Qpid Dispatch *R*outer network

or

* displays information about a Qpid Dispatch *r*outer network?

I am currently thinking that lowercase r is better choice here.

Reply:
to be consistent with the docs you'd use a capital R here.
An alternative would be to reword to:

* displays information about a router network

which, to my eye, looks better. No need to explicitly state the 'long name' everywhere. wdyt?

Reply:
Thanks, I'll go with capital R then.